shared/chat/conversation/messages/wrapper/send-indicator/index.tsx

```diff
@@ -32,7 +32,24 @@ class SendIndicator extends React.Component<Props, State> {
 
   constructor(props: Props) {
     super(props)
-    this.state = {iconStatus: 'encrypting', visible: !props.sent}
+    const state: State = {iconStatus: 'encrypting', visible: !props.sent}
+
+    if (!(this.props.sent || this.props.failed)) {
```

Comment:
`props` instead of `this.props`

Reply:
🐑 I think this is easier to understand if we flip it a la De Morgan's. `!this.props.sent && !this.props.failed`

Reply:
this is just a movement of the below. also here props === this.props